intro/Readme.md

```diff
@@ -0,0 +1,60 @@
+### 1) Provide a link in your Readme.md that links to the documentation for the following classes:
```

Comment:
Not sure if I clearly understood the question, but if I just need to remove links I would do updates in Readme.md file and push it to the branch. 
I do not have changes on master and as I understand, I do not need to rebase branch to the master.
???